shopinvader/views/sale_view.xml

```diff
@@ -86,12 +86,4 @@
         </field>
     </record>
 
-    <record id="sale.action_quotations" model="ir.actions.act_window">
-        <field name="domain">[('typology', '=', 'sale')]</field>
```

Comment:
To solve the compatibility with shopinvader_quotation, maybe the best is simply to change the domain to [('typology', '!=', 'cart')] ?

In long term we should discuss about the UI, and define the menu correctly